### DISCUSSION TITLE
Icon Request: fa-firefighter

### DISCUSSION THREAD

**Discussion by aethior** [FortAwesome/Font-Awesome#21227] [Icon Requests]
There is a medic icon, but there aren't a police (https://github.com/FortAwesome/Font-Awesome/issues/3372) or firefighter icons.
Very usefull icons for government sites and public service sites.

![firefighter_1f9d1-200d-1f692](https://user-images.githubusercontent.com/1769227/164948170-6ae767e5-a828-4f03-b7a6-9e1832faa63e.png)

*Related Unicode Emoji* #222
👨‍🚒 / 👩‍🚒 [U+1F9D1,‍ U+200D, U+1F692] (https://emojipedia.org/firefighter/) Firefighter

**Comment by MadTiago** [comment]
+1

**Comment by dpfem** [comment]
+1

**Comment by ronifabio** [comment]
+1

**Comment by iShaymus** [comment]
+10

**Comment by rhanka** [comment]
+1

**Comment by brettjbush** [comment]
+1

**Comment by lrdiv** [comment]
+1

**Comment by katleiahramos** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
